packages/workbench/README.md

```diff
@@ -0,0 +1,7 @@
+# Theia - Workbench Extension
+
+See [here](https://github.com/theia-ide/theia) for a detailed documentation.
```

Comment:
The documentation should link to https://www.theia-ide.org/doc/index.html instead of https://github.com/theia-ide/theia. See here https://github.com/theia-ide/theia/pull/2925#discussion_r219389249.

Reply:
I copied it from the existing `workspace` extension. The majority of the extensions in Theia have the same link. Would it be better to update them all in one go rather than have this one different?

Reply:
> Would it be better to update them all in one go rather than have this one different?

Please open a follow-up.

Reply:
https://github.com/theia-ide/theia/issues/3031